client/src/components/Login.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+    <div id="login">
+        <h1>Login</h1>
+        <p>Sign in to track your project progress!</p>
+        <label for="username">Email</label>
+        <input id='username' type="text" name="username" v-model="input.username" placeholder="joebruin@g.ucla.edu" />
+        <label for="password">Password</label>
+        <input id='password' type="text" name="password" v-model="input.password" placeholder="Password" />
+        <button type="button" v-on:click="login()">Login</button>
+    </div>
+</template>
+
+<script>
+    export default {
+        name: 'Login',
+        data() {
+            return {
+                input: {
+                    username: "",
+                    password: ""
+                }
+            }
+        },
+        methods: {
+            login() {
+                if(this.input.username != "" && this.input.password != "") {
+                    if(this.input.username == this.$parent.mockAccount.username && this.input.password == this.$parent.mockAccount.password) {
```

Comment:
Try to make this into a separate function call so that you can easily change it when you actually start linking with the API.